client/lib/titan/get-titan-urls.js

```diff
@@ -25,14 +25,14 @@ function getTitanUrl( email, app = TITAN_APPS.EMAIL ) {
 	return titanAppUrl.href;
 }
 
-export function getTitanEmailUrl( email ) {
+export function getTitanCalendarlUrl( email ) {
 	return getTitanUrl( email, TITAN_APPS.EMAIL );
```

Comment:
Shouldn't this be `TITAN_APPS.CALENDAR`? (Same applies below for the contacts URL.)